go.mod

```diff
@@ -6,8 +6,9 @@ toolchain go1.24.1
 
 require (
 	github.com/ClickHouse/clickhouse-go/v2 v2.32.2
-	github.com/aquasecurity/trivy v0.60.0
-	github.com/castai/image-analyzer v0.7.2
+	github.com/aquasecurity/testdocker v0.0.0-20240730052646-35e4cd04890c
+	github.com/aquasecurity/trivy v0.61.0
+	github.com/castai/image-analyzer v0.7.4-0.20250508113458-eca6aeb1f4fd
```

Comment:
Use branch until https://github.com/castai/image-analyzer/pull/53 is merged and release